IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DoorTypeTurnstile/DocPropertySet.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_DoorTypeTurnstile" UniqueId="c5233710-61f4-8eb7-185b-60b03c52ca80" ApplicableType="IfcDoor/TURNSTILE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="IsBidirectional_3PYJ6a4gXACx94JYuDo4no" />
+		<DocProperty xsi:nil="true" href="GateID_1vaewDvSn2zgrPRZgB_Hg9" />
+		<DocProperty xsi:nil="true" href="PassengerThroughput_0bZXuIvtrB3QZWBy_YgRO2" />
+		<DocProperty xsi:nil="true" href="TicketProcessingTime_17M9Cc_kz818DfWA4GZJTA" />
+		<DocProperty xsi:nil="true" href="TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK" />
```

Comment:
@czapplitec, I Understand these are important properties, but are they common amoung all turnstiles or do they deserve a seperate PSet so they can be used on other door types e.g. boom barrier, which often has a ticketting component?? thoughts? 
Additionally if these are for monitoring performance history they should be in a performance PSet, whereas if they are performance ratings provided by the manufacturer my above comment stands.

Reply:
These are not for performance history, just fixed information from manufacturers

Reply:
Updated this part.